src/components/Item.js

```diff
@@ -0,0 +1,40 @@
+import React, { useState, useEffect } from 'react';
+import '../styles/Item.css';
+
+const Item = ({ item, handleChange }) => {
+  const [checked, setChecked] = useState(false);
+  const className = checked ? 'completed' : '';
+
+  useEffect(() => {
+    const checkDate = () => {
+      // Current date in seconds
+      const now = Date.now() / 1000;
+      // Number of seconds in a day
+      const oneDay = 60 * 60 * 24;
+      // Checks to see if the number of seconds that have elapsed since
+      // the last purchased date is less than one day
+      const isChecked = now - item.lastPurchasedDate.seconds < oneDay;
+      return isChecked;
+    };
+    // check whether item should be checked
+    const check = checkDate(item);
+    setChecked(check);
```

Comment:
Just a question: Do items need to be checked when they are created? Right now when I add a new item it is checked and has a strikethrough.

Reply:
The strikethrough on creation isn't required per the AC, but does make sense with how the app should work IMO. Check out this comment for some more deets: https://github.com/the-collab-lab/tcl-8-smart-shopping-list/pull/29#discussion_r446485946